dev-env/etc/nix.conf

```diff
@@ -1,11 +1,11 @@
 build-max-jobs = 2
 
-extra-substituters = https://nix-cache.da-ext.net
+substituters = https://nix-cache.da-ext.net https://cache.nixos.org
```

Comment:
I think `extra-substituters` is actually fine, at least it doesn’t produce a warning. But fine to drop it too.

Reply:
Looks like it, but I'd rather not have to explain why we include the NixOS cache keys without including the cache itself.